get-latest-tgf.ps1

```diff
@@ -0,0 +1,40 @@
+$ErrorActionPreference = "Stop" #Make all errors terminating
+
+try {
+    $TGF_PATH = (Get-Command tgf.exe).Path
+    Write-Host tgf path: $TGF_PATH
+    $LOCAL_VERSION = (tgf.exe --current-version) | Out-String
+    $LOCAL_VERSION = [regex]::Matches($LOCAL_VERSION,'\d+\.\d+\.\d+').Value
+} catch {
+    $TGF_PATH = Join-Path -Path (Get-Location) -ChildPath "tgf.exe"
+    Write-Host tgf not found. using $TGF_PATH
+} finally {
+    Write-Host "- tgf version (local):" $LOCAL_VERSION
```

Comment:
If tgf is not get installed, you will get nothing here. Not sure if it is intended. If not, the Write-Host should be moved in the first section and get rid of the finally block.

Reply:
Yeah it was intended